app/models/manageiq/providers/kubernetes/container_manager/scanning/job.rb

```diff
@@ -395,4 +398,13 @@ def add_secret_to_pod_def(pod_def, inspector_admin_secret_name)
   def inspector_image
     'docker.io/openshift/image-inspector:2.0'
   end
+
+  def inspector_proxy_env_variables
+    settings = ext_management_system.custom_attributes
+    PROXY_ENV_VARIABLES.each_with_object([]) do |st, env|
```

Comment:
> no_proxy means a list of hostnames that don't need any proxy.

Hrm, I'm still confused by this.

So, the custom attributes will have a list of hosts associated to each env variable?  In other words:

```ruby
  no_proxy = ["host1.org", "host2.org", host3.org"]
  https_proxy = ["host4.org", "host5.org", "host6.org"]
```

Something like that?